rook/base/ceph-hdd/cephobjectstore.yaml

```
@@ -0,0 +1,62 @@
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: ceph-hdd-object-store
+  namespace: ceph-hdd
+  annotations:
+    argocd.argoproj.io/sync-wave: "2"
+spec:
+  metadataPool:
+    failureDomain: zone
+    replicated:
+      size: 3
+  dataPool:
+    failureDomain: zone
+    replicated:
+      size: 3
+  preservePoolsOnDelete: true
+  gateway:
+    type: s3
+    sslCertificateRef:
+    port: 80
+    securePort:
+    instances: 1
+    # A key/value list of annotations
+    annotations:
+    #  key: value
```

I'm not sure this is valid YAML...  What about
```suggestion
    annotations: {}
```

---

This field can be omitted, so removed it.
And other fields (sslCertificateRef, securePort, placement) are removed as well.